skiko/src/jsMain/kotlin/org/jetbrains/skiko/RenderFactory.js.kt

```diff
@@ -1,14 +1,9 @@
 package org.jetbrains.skiko
 
-import org.jetbrains.skiko.context.*
 import org.jetbrains.skiko.redrawer.*
 
 internal actual fun makeDefaultRenderFactory(): RenderFactory {
     return object : RenderFactory {
-        override fun createContextHandler(layer: SkiaLayer, renderApi: GraphicsApi): ContextHandler {
-            TODO()
-        }
-
         override fun createRedrawer(
```

Comment:
Seems `makeDefaultRenderFactory()` commonization was premature, and it better be removed or de-commonized.

Reply:
Maybe we do it in another PR? We probably should decommonize RenderFactory, SkiaLayerProperties, and PictureHolder

Reply:
OK